contracts/FlatLaunchpeg.sol

```diff
@@ -107,7 +109,9 @@ contract FlatLaunchpeg is BaseLaunchpeg, IFlatLaunchpeg {
         }
         allowList[msg.sender] -= _quantity;
         uint256 totalCost = mintlistPrice * _quantity;
+
         _mint(msg.sender, _quantity, "", false);
+        amountMintedDuringMintlist += _quantity;
```

Comment:
Should we make sure in this function that it's not possible to mint more than `amountForMintlist` as we do in Launchpeg?

```solidity
if (amountMintedDuringMintlist + _quantity > amountForMintlist) {
    revert Launchpeg__MaxSupplyReached();
}
```